src/tiered/tiered_work.c

```diff
@@ -90,6 +91,36 @@ __wt_tiered_push_work(WT_SESSION_IMPL *session, WT_TIERED_WORK_UNIT *entry)
     return;
 }
 
+/*
+ * __wt_tiered_push_work --
+ *     Push a work unit to the queue. Assumes it was previously returned from one of the get
+ *     functions, and it is being re-queued.
+ */
+void
+__wt_tiered_push_work(WT_SESSION_IMPL *session, WT_TIERED_WORK_UNIT *entry)
```

Comment:
Would it make sense to rename this function to make it clear that this is for work that is being re-queued? (I'm not sure it's actually necessary, but I felt like it's at least worth the question.)

Reply:
Agreed.  Renamed as `__wt_tiered_requeue_work`.